src/node_metadata.h

```diff
@@ -38,6 +38,8 @@ namespace node {
   V(nghttp2)                                                                   \
   V(napi)                                                                      \
   V(llhttp)                                                                    \
+  V(undici)                                                                    \
+  V(acorn)
```

Comment:
I think the backslash is missing here, right?
```suggestion
  V(undici)                                                                    \
  V(acorn)                                                                     \
```

Reply:
Dont think its compulsory since it would be the last line
but anyways adding it for good measure

Reply:
is it best practice to add it even for the last line? i am unsure

Reply:
I think we should keep it as it was there before, unless there's a good reason to get rid of it.

Reply:
done!

Reply:
Looks like my suggestion breaks the C++ linter, and that's a good reason not to apply it 😅 sorry!